apps/client/src/app/career-qna/page.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import styled from '@emotion/styled';
+import { colors } from '@sickgyun/design-token';
+import { Flex, Stack, Text } from '@sickgyun/ui';
+import LeftArrowIcon from '@sickgyun/ui/src/Icons/LeftArrow';
+import RightArrowIcon from '@sickgyun/ui/src/Icons/RightArrow';
+import { useState } from 'react';
+import Footer from '@/components/common/Footer';
+import Header from '@/components/common/Header';
+import QnaBox from '@/components/qna-posting/QnaBox';
+import QnaListBox from '@/components/qna-posting/QnaListBox';
+import QnaPostList from '@/components/qna-posting/QnaPostList';
+import { QNA } from '@/constants/qna';
+
+const CareerQna = () => {
+  const [currentQnaPageIndex, setCurrentQnaPageIndex] = useState(0);
+  const [leftArrowIconColor, setLeftArrowIconColor] = useState<string>(colors.gray600);
+  const [rightArrowIconColor, setRightArrowIconColor] = useState<string>(colors.gray600);
```

Comment:
```suggestion
  const [leftArrowIconColor, setLeftArrowIconColor] = useState(colors.gray600);
  const [rightArrowIconColor, setRightArrowIconColor] = useState(colors.gray600);
```

string으로 추론하기 때문에 불피요한 제네릭 타입이라고 생각되는데 따로 넣으신 이유가 있을까요?

Reply:
'"#BBBCC2"' 형식의 인수는 'SetStateAction<"#70737C">' 형식의 매개 변수에 할당될 수 없습니다.
<string>을 써주지 않으면 이렇게 뜨더라구요 그래서 사용했습니다.